src/Path.php

```diff
@@ -27,13 +27,22 @@ public function relativize(string $absolutePath): string
     {
         $relativePath = str_replace($this->workingDirectory, '.', $absolutePath);
 
-        return is_dir($absolutePath) ? rtrim($relativePath, '/').'/' : $relativePath;
+        return is_dir($absolutePath) ? rtrim($relativePath, DIRECTORY_SEPARATOR).DIRECTORY_SEPARATOR : $relativePath;
```

Comment:
could be more portable with `rtrim($relativePath, '/'.DIRECTORY_SEPARATOR).DIRECTORY_SEPARATOR`